app/components/auth/signUp/components/surveyForm/index.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { Dispatch } from 'redux';
+import { connect } from 'react-redux';
+import { shuffle } from 'lodash';
+import { withStyles } from '../../../../../helpers/withStylesHelper';
+import { SCINAPSE_SURVEY_QUESTIONS, Survey, AnswerToQuestion } from './constants';
+import { AppState } from '../../../../../reducers';
+import { getCurrentPageType } from '../../../../locationListener';
+import { ActionCreators } from '../../../../../actions/actionTypes';
+import Question from './components/question';
+import ActionTicketManager from '../../../../../helpers/actionTicketManager';
+import GlobalDialogManager from '../../../../../helpers/globalDialogManager';
+import { SurveyTicketFormatter } from './helpers/SurveyTicketManager';
+const styles = require('./surveyForm.scss');
+
+type Props = ReturnType<typeof mapStateToProps> & { dispatch: Dispatch<any> };
+
+function randomizedAnswers(defaultQuestion: Survey) {
+  const defaultAnswers = defaultQuestion.answers;
+  const randomizedAnswersSurveyContext = {
+    ...defaultQuestion,
+    answer: shuffle(defaultAnswers),
+  };
+  return randomizedAnswersSurveyContext;
+}
+
+function openFinalSignUpDialog(nextSignUpStep: string) {
+  if (nextSignUpStep === 'email') {
+    GlobalDialogManager.openFinalSignUpWithEmailDialog();
+  } else {
+    GlobalDialogManager.openFinalSignUpWithSocialDialog();
+  }
+}
+
+function getAllSurveyName() {
```

Comment:
I expect `string` when the function name is ended with `name`.
how about using `getAllSurveys`